themes/openy_themes/openy_carnation/src/js/openy_carnation.js

```diff
@@ -0,0 +1,460 @@
+/**
+ * @file
+ * OpenY Carnation JS.
+ */
+(function ($) {
+
+  "use strict";
+  Drupal.openyCarnation = Drupal.openyCarnation || {};
+
+  /**
+   * Alert Modals
+   */
+  Drupal.behaviors.openyAlertModals = {
+    attach: function (context, settings) {
+      var alertModals = $('.alert-modal', context);
+
+      $(window).on('load',function() {
+        if (alertModals.length) {
+          alertModals.modal('show');
+        }
+      });
+    }
+  };
+
+  /**
+   * Move Header Banner paragraph to header.
+   */
+  Drupal.behaviors.openyBanners = {
+    attach: function (context, settings) {
+      var bannerHeader = $('.paragraph--type--banner, .landing-header, .page-heading');
+      if (bannerHeader.length > 0) {
+        $('.banner-zone-node').once('openy-banners').append(bannerHeader.eq(0));
```

Comment:
I don't like this approach of moving pieces of HTML around. Let's discuss this approach. We should be able to accomplish it with templates.

Reply:
Honestly, I hate using jQuery to move elements in the DOM around as well... Problem lives in OpenY default install profile. The "header" region attached to the various content types allows the user to place any type of paragraph type into its region. 

Without using a bunch of preprocess functions to pull out just the banner & gallery paragraph types outside of the {{ page.header }} region I'm not too sure we can accomplish this.